.travis.yml

```diff
@@ -16,13 +16,21 @@ matrix:
       os: osx
       osx_image: xcode8.3
     - script: make docker_test
-      env: JOB=Linux3.1
+      env: JOB=Linux-Swift3.1
       sudo: required
       services: docker
-    - script: make docker_test_302
-      env: JOB=Linux3.0.2
+    - script: make docker_test_4
+      env: JOB=Linux-Swift4
       sudo: required
       services: docker
+    - script: swift test
+      env: JOB=Linux-Swift4-without-Docker
+      sudo: required
+      dist: trusty
+      install:
+        - export SWIFT_VERSION=swift-4.0-DEVELOPMENT-SNAPSHOT-2017-05-11-a
+        - curl https://swift.org/builds/swift-4.0-branch/ubuntu1404/$SWIFT_VERSION/$SWIFT_VERSION-ubuntu14.04.tar.gz|tar zx -C /
```

Comment:
I would like to be able to use it without docker, including resolving dependencies, without setting additional environment variables.